firefox-ios/Providers/Profile.swift

```diff
@@ -598,6 +598,8 @@ open class BrowserProfile: Profile {
                     case .tabReceived(_, let tabData):
                         let url = tabData.entries.last?.url ?? ""
                         return URL(string: url, invalidCharacters: false)
+                    case .tabsClosed:
+                        return nil
```

Comment:
@OrlaM tagging you since it seems you worked in this area previously.
There seems to be a new case for `IncomingDeviceCommand` called `tabsClosed`, I added new code so that the changes don't take affect, but should we discuss with the team that implemented these changes? 

Also, do we want to create any tickets to handle this in the future?

Reply:
Hi @cyndichin! I landed the new command in Application Services ☺️ Thanks so much for fixing this up, and sorry for the breakage! Your patch looks great to me; it's safe for iOS to ignore the `tabsClosed` command, as well as any other incoming device commands it doesn't understand.

Reply:
awesome, thanks @linabutler for providing the context and details! Do you mind attaching the PR regarding the change from application services side?

Reply:
Here you go! https://github.com/mozilla/application-services/pull/6182

Reply:
thanks!